gems/actionpack/6.0/actionpack-generated.rbs

```diff
@@ -2748,9 +2736,6 @@ module ActionController
 
     def render: (*untyped args) -> untyped
 
-    # Overwrite render_to_string because body can now be set to a Rack body.
-    def render_to_string: () -> untyped
```

Comment:
Could you move this entry instead of removal?

ref: https://github.com/rails/rails/blob/v8.0.2/actionpack/lib/action_controller/metal/rendering.rb#L174

Reply:
Addressed in this commit 3f1ba37.